Sources/Diagnostics/DiagnosticsTracker.swift

```diff
@@ -159,14 +159,12 @@ private extension DiagnosticsTracker {
     func clearDiagnosticsFileIfTooBig() async {
         if await self.diagnosticsFileHandler.isDiagnosticsFileTooBig() {
             await self.diagnosticsFileHandler.emptyDiagnosticsFile()
-            self.trackMaxEventsStoredLimitReached()
+            await self.clearDiagnosticsFileIfTooBig()
```

Comment:
Hmm is this needed? I guess when calling this, since we just emptied the file in the line above, it shouldn't do anything, but shouldn't be needed normally?

Reply:
not needed right. I forgot to remove this line when copying the content of `track`